File: 2-Threads/2024/python/numeros.py
import random

class Numeros:
    """Classe com métodos para manipulação de listas de números.
    """
    @staticmethod
    def popular_aleatorio(lista, quantidade, nome_lista):
        """Método para popular uma lista com números aleatórios.

        Args:
            lista (int): Lista a ser populada.
            quantidade (int): Quantidade de números a serem gerados.
            nome_lista (string): Nome da lista a ser populada para exibição.
        """
        for _ in range(quantidade):
            lista.append(random.randint(0, 1000))
        print(f"Lista {nome_lista} populada com {quantidade} números aleatórios.")

File: 2-Threads/2024/cs/Numeros.cs
using System;
using System.Collections.Generic;
class Numeros
{
    public static void PopularAleatorio(List<int> lista, int quantidade, string nomeLista)
    {
        Random random = new Random();
        for (int i = 0; i < quantidade; i++)
        {
            lista.Add(random.Next());
        }
        Console.WriteLine($"Lista {nomeLista} populada com {quantidade} números aleatórios.");
    }
}